packages/vue-components/src/Overlay.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <div>
+    <span
```

Comment:
suggestion: pull the icon upward using scoped slots, so the component specific styles can be moved up there

not sure if it would end up more verbose though 😐

Reply:
Shifted this up. Doesn't look too verbose :)